### ISSUE TITLE
Can I generate mathematical expression text images?

### ISSUE THREAD

**Issue by ai-motive** [Belval/TextRecognitionDataGenerator#183]
such as 
![스크린샷, 2020-12-04 12-42-46](https://user-images.githubusercontent.com/69468134/101119226-3de8e200-362e-11eb-969a-b20c2689f0f9.png)

**Comment by Belval** [comment]
Hi! Sorry for the late response but I don't think so, at least not without some serious tweaks to the generator. 

It would be a nice feature though so I will leave the issue open.

**Comment by ai-motive** [comment]
Thanks for responding ~ I think you need a function to print latex text as an image ~

**Comment by abdur75648** [comment]
Yes @ai-motive, I had done this by rendering latex equations as images.
@Belval Should I add it as a PR here?
